test-common/src/lib.rs

```diff
@@ -23,29 +24,53 @@ impl PostgresTestHarness {
     const DB: &'static str = "postgres-db-test";
     const USER: &'static str = "postgres-user-test";
     const PASSWORD: &'static str = "postgres-password-test";
+    #[allow(dead_code)]
+    const IMAGE_URI_MASTER_BRANCH_14: &'static str =
+        "ghcr.io/timescale/dev_promscale_extension:master-ts2-pg14";
+    #[allow(dead_code)]
+    const IMAGE_URI_LOCAL_14: &'static str = "local/dev_promscale_extension:head-ts2-pg14";
```

Comment:
Do you need these constants for the upgrade tests or are they all for the sake of completeness?

Reply:
I think so. Either an env var will specify the image to use, OR we'll run with the default. The default is set to 14. If the env var is set to a v13 or v12, I need to detect it and use a "latest" image of the same version.

Reply:
I have a suggestion then. 

Firstly, you'd need to introduce two enums: 
```rust
enum ImageOrigin {
  Local,
  Master,
}

enum PgVersion {
  v14, v13, v12
}
```

Then you can have a function that returns an image URI based on these two parameters:
```rust
fn pg_image_uri(origin: ImageOrigin, version: PgVersion) -> String
```

Reply:
I like that. Updated